e2e/client.test.js

```diff
@@ -0,0 +1,451 @@
+    cwd: `${PATH}/server`
+  })
+    .toString()
+    .trim()
+  apiContext = await playwright.request.newContext({
+    baseURL: 'http://localhost:4466',
+    extraHTTPHeaders: {
+      Authorization: token,
+      'faq-tenant': 'default/default'
+    }
+  })
+  prisma = multiTenant.current({
+    headers: {
+      'faq-tenant': 'default/default'
+    }
+  })
+  config = await upsertConfigMutation(apiContext)
+  prisma._meta = { ...prisma._meta, configuration: config.upsertConfiguration }
+  user = await createUserMutation(apiContext)
+  tags = await tagsIdQuery(apiContext)
+})
+
+test.beforeEach(async () => {
+  const deleteCommands = [
+    'deleteManyAnswers',
+    'deleteManyQuestions',
+    'deleteManyHistoryActions',
+    'deleteManyFlags',
+    'deleteManyTags',
+    'deleteManyZNodes'
+  ]
+  for (const command of deleteCommands) {
+    await apiContext.post('/', {
+      data: {
+        query: deleteAll(command)
+      }
+    })
+  }
+})
+
+test('Shoud be able to create a question', async ({ page }) => {
+  await page.goto('/')
+  await page
+    .locator('button', { hasText: 'Nouvelle question' })
+    .first()
+    .click()
+  await page.locator('input').click()
+  await page.locator('input').fill(questionsText[randomQuestion])
+  await page.getByRole('button', { name: 'add' }).click()
+  await page
+    .getByText(tags.tagName, { exact: true })
+    .first()
+    .click()
+  await page.locator('button', { hasText: 'Envoyer la question' }).click()
+  await expect(page.getByRole('heading', { name: questionsText[randomQuestion] })).toBeVisible()
+})
+
+test('Should be able to create a question and answer it', async ({ page }) => {
+  await page.goto('/')
+  await page
+    .locator('button', { hasText: 'Nouvelle question' })
+    .first()
+    .click()
+  await page.locator('input[type=text]').click()
+  await page.locator('input[type=text]').fill(questionsText[randomQuestion])
+  await page.getByRole('button', { name: 'add' }).click()
+  await page
+    .getByText(tags.tagName, { exact: true })
+    .first()
+    .click()
+  await page.locator('button', { hasText: 'Envoyer la question' }).click()
+  await expect(page.getByRole('heading', { name: questionsText[randomQuestion] })).toBeVisible()
+  await page.locator('button', { hasText: 'Répondre à la question' }).click()
+  await page.locator('textarea').click()
+  await page.locator('textarea').fill(answersText[randomAnswer])
+  await page.locator('button', { hasText: 'Envoyer la réponse' }).click()
+  await expect(page.getByText(answersText[randomAnswer], { exact: true })).toBeVisible()
+})
+
+test('Should return a search result', async ({ page }) => {
+  const zNode = await prisma.mutation.createZNode(createZNodeParams(tags.tagId, user))
+  await algolia.addNode({ prisma }, zNode.id)
+  await page.goto('/')
+  await page.locator('input[type=text]').click()
+  const slicedQuestion = questionsText[randomQuestion].slice(0, 4)
```

Comment:
Je suggère ce nom qui me semble plus clair (idem pour les autres occurences)
```suggestion
  const searchQuery = questionsText[randomQuestion].slice(0, 4)
```